core/src/ast_builder/create_table.rs

```diff
@@ -1,5 +1,9 @@
 use {
     super::Build,
+    crate::ast_builder::column_def::PrimaryKeyConstraintNode,
+    crate::error::TranslateError,
+    crate::parse_sql::parse_column_def,
+    crate::translate::translate_column_def,
     crate::{ast::Statement, ast_builder::ColumnDefNode, result::Result},
```

Comment:
nit. let's merge this into a single `crate::{ .. }`

Reply:
Is there a configuration of cargo fmt that can do that automatically or do I need to do that by hand?

Reply:
as i know that this kind of merging use fmt option is only being supported in rust nightly. so.. I need to check it by myself.

Reply:
I have no issue switching to the nightly build to run a fmt, do let me know if you can find the appropriate format option. It would surely massively speed up this process.

Reply:
the last time I checked was may be about.. 3 years ago. so I dont remember that. could i get this from you if you ok..?

Reply:
Ok found it, I have also created a `rustfmt.toml` defining this configuration so that it will be standardized.

Reply:
Done in commit 9b75794

Reply:
Done, manually, in commit [9c47a24](https://github.com/gluesql/gluesql/pull/1515/commits/9c47a24e02fb457ea12d679eaabf46bad2a60525) and 
9f24309